sympy/core/symbol.py

```diff
@@ -205,10 +205,47 @@ def numbered_string_incr(s, start=0):
 
 class Symbol(AtomicExpr, Boolean):
     """
+    A Symbol is a representation of a mathematical variable using a
+    name. It supports subscripts and Greek and Latin characters.
+    Symbols can be combined to form SymPy expressions. A single
+    Symbol can be created using the Symbol() constructor but to create
+    multiple symbols in one statement use the symbols() function.
+
     Assumptions:
        commutative = True
+       algebraic = None
+       antihermitian = None
+       complex = None
+       composite = None
+       even = None
```

Comment:
I don't think it's useful to just list all of these. They are described elsewhere so a link to somewhere else would be better if we want to show the full list of assumptions predicates:
https://docs.sympy.org/latest/guides/assumptions.html#predicates

Reply:
Thanks. I have made these changes